index.html

```diff
@@ -102,9 +103,9 @@ <h3 class="h5">Kari Nordmann</h3>
         </div>
       </div>
     </section>
-    <section class="d-flex align-items-center justify-content-center">
-      <div class="card mb-2 row col-10">
-        <div class="border d-flex flex-row">
+    <section class="d-flex align-items-center justify-content-center row">
```

Comment:
```suggestion
    <section class="align-items-center justify-content-center row">
```

Row is d-flex by default